manifests/server/database.pp

```diff
@@ -15,7 +15,7 @@
   Optional[String[1]]                           $updateref       = undef,
   Optional[String[1]]                           $limits          = undef,
   # BDB/HDB options
-  Optional[String[1]]                           $dboptions       = undef,
+  Optional[Hash]                                $dboptions       = undef,
```

Comment:
Maybe we can check that both the key and value are non-empty strings (untested, and the test suite obviously does not test this)?

```suggestion
  Optional[Hash[String[1],String[1]]]           $dboptions       = undef,
```